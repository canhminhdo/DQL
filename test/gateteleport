#!/bin/sh

DIR=../test
cd ../examples && maude < gateteleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/gateteleport.out
diff $DIR/gateteleport.expected $DIR/gateteleport.out
rm $DIR/gateteleport.out