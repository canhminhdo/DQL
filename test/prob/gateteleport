#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < gateteleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/gateteleport.out
diff $DIR/gateteleport.expected $DIR/gateteleport.out
rm $DIR/gateteleport.out