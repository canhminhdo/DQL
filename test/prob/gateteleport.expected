==========================================
reduce in GATE-TELEPORT : initQState |= [H(1) ; CX(1,2) ; H(3) ; CX(3,4) ; CX(
    3,2) ; CX(0,1) ; H(0) ; CX(4,5) ; H(4) ; if P(0,|0>) then skip else Z(2) ;
    Z(3) fi ; if P(1,|0>) then skip else X(2) fi ; if P(5,|0>) then skip else
    X(2) ; X(3) fi ; if P(4,|0>) then skip else Z(3) fi](P(2,3,qubitAt(
    targetQState, 0 1)) with prob == 1) .
result JudgmentSet: emptyJS
Bye.
